src/adapters/demo-result.mapper.ts

```diff
@@ -0,0 +1,45 @@
+import { EmpathyResult } from '@empathy/search-adapter';
+import { Result } from '@empathy/search-types';
+
+type CustomResult = {
```

Comment:
We agreed in today's standup to use `module augmentation` in our `Result` model so we can access these properties from anywhere in the codebase.

Can you change this approach and use `module augmentation` instead? If you have any doubt just hit me up, as I've tried it already and works fine.